doc/deployment/connect-to-cluster.md

```diff
@@ -0,0 +1,187 @@
+
+To connect to the Pachyderm dashboard, you can either use port-forwarding,
+or the IP address of the virtual machine on which your Kubernetes cluster
+is running.
+
+The following example shows how to access a Pachyderm cluster
+that runs in `minikube`.
+
+To connect to a Pachyderm dashboard, complete the following steps:
+
+* To use port-forwarding, run:
+
+  ```bash
+  pachctl port-forward
+  ```
+
+* To use the IP address of the VM:`
+
+  1. Get the minikube IP address:
+
+     ```bash
+     $ minikube ip
+     ```
+
+  1. Point your browser to the following address:
+
+     ```bash
+     <minikube_ip>:30080
+     ```
+
+## Connect to a Cluster Deployed on a Cloud Platform
+
+Similarly to a local cluster deployment, in Pachyderm
+cluster deployed on a cloud platform, port-forwarding
+from `pachctl` to `pachd` is enabled by default and
+to access the Pachyderm dashboard, you need to run
+`pachctl` port-forward`. Since a Pachyderm cluster
+deployed on a cloud platform is more likely to become
+a production deployment, using the Pachyderm contexts
+configuring a `pachd_address` as described in
+[Connect by using a Pachyderm context](3connect-by-using-a-pachyderm-context.html)
+is a preferred way.
+
+## Connect by using a Pachyderm context
+
+You can specify an IP address that you use to connect to the
+Pachyderm UI and the S3 gateway by storing that address in the
+Pachyderm configuration file as the `pachd_address` parameter.
+If you have already deployed a Pachyderm cluster, you can
+set a Pachyderm IP address by updating your cluster configuration
+file.
+
+This configuration is supported for deployments that do not have
+a firewall set up between the Pachyderm cluster deployed in the cloud
+and the client machine.
+Defining a dedicated `pachd` IP address and host is a more reliable
+way that might also result in better performance compared to
+port-forwarding. Therefore, Pachyderm
+recommends that you use contexts in all production environments.
+
+This configuration requires that you deploy an ingress controller
+and a reliable security method to protect the traffic between the
+Pachyderm pods and your client machine. Remember that exposing your
+traffic through a public ingress might
+create a security issue. Therefore, if you expose your `pachd` endpoint,
+you need to make sure that you take steps to protect the endpoint and
+traffic against common container security threats. Port-forwarding
+might be an alternative which might result in sufficient performance
+if you place the data that is consumed by your pipeline in object
+store buckets located in the same region.
+
+For more information about Pachyderm contexts, see
+[Manage Cluster Access](../managing_pachyderm/connect-to-cluster.html)
+
+To connect by using a Pachyderm context, complete the following
+steps:
+
+1. Get the current context:
+
+   ```bash
+   $ pachctl config get active-context
+   ```
+
```

Comment:
Suggest we add

This returns the name of the currently active context.  Use this as the argument <name> to the command below.